NAMESPACE

```diff
@@ -55,10 +55,15 @@ export(simulate_secondary)
 export(simulate_uniform_cases)
 export(summarise_draws)
 export(summarise_variable)
+export(validate_latent_individual)
```

Comment:
We might want an issue to make this an s3 method - that way epdist.default can call a validation function on its input to check its a implemented model

Reply:
Nice idea. I'll make an issue.